_data/photos.yml

```diff
@@ -3,227 +3,227 @@
   file: "wtmvie-1.jpg"
   text: "elsurneb Art & Photography"
   link: "https://elsurneb.com"
-  year: "2018"
+  year: "2019"
```

Comment:
This denotes the year of when a picture has been taken.
Even if we plan the 2019 event it still will be a 2018 picture :)